wheel/python/clvm_rs/at.py

```diff
@@ -0,0 +1,40 @@
+from __future__ import annotations
+from typing import Optional, cast
+
+from .clvm_storage import CLVMStorage
+
+
+def at(obj: CLVMStorage, position: str) -> Optional[CLVMStorage]:
+    """
+    Take a string of `f` and `r` characters and follow that path.
+
+    Example:
+
+    ```
+    p1 = Program.to([10, 20, 30, [15, 17], 40, 50])
+    assert Program.to(17) == at(p1, "rrrfrf")
+    ```
+
+    Returns `None` if an atom is hit at some intermediate node.
+
+    ```
+    p1 = Program.to(10)
+    assert None == at(p1, "rr")
+    ```
+
+    """
+    v: Optional[CLVMStorage] = obj
+    for c in position.lower():
+        v = cast(CLVMStorage, v)
```

Comment:
is this just to cast-away the optionality? It seems redundant

Reply:
It's just to make `mypy` happy.

Reply:
Google "mypy using `cast`", for example https://adamj.eu/tech/2021/07/06/python-type-hints-how-to-use-typing-cast/ or https://stackoverflow.com/questions/55461519/python-type-hints-better-syntax-for-cast

Reply:
Actually, you're right. This was required until I made some other change, so I'll get rid of it.